### ISSUE TITLE
[css-blocks] synthetic conflicts

### ISSUE THREAD

**Issue by chriseppstein** [linkedin/css-blocks#51]
There should be a way to create resolution constraints that span properties that don't naturally conflict in css. This can be used to implement things like accessibility constraints for contrast between foreground and background colors.

Such a capability should be extensible within an application or css library that is built on css-blocks.
